qiskit_research/utils/convenience.py

```diff
@@ -38,14 +38,19 @@ def add_dynamical_decoupling(
     dd_str: str,
     scheduler: BaseScheduler = ALAPScheduleAnalysis,
     add_pulse_cals: bool = False,
+    urdd_pulse_num: int = 4,
```

Comment:
Why did you choose 4 as the default?

Reply:
The minimum number of pulses for a URDD sequence is 4 (and it must be even). This is now reflected in the docstrings.